packages/animations/lib/src/open_container.dart

```diff
@@ -246,6 +248,9 @@ class OpenContainer<T extends Object> extends StatefulWidget {
   /// to the nearest navigator.
   final bool useRootNavigator;
 
+  /// By default, [opaque] is true
```

Comment:
This needs to document what opaque does.

Reply:
Done, please have a look.